pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -99,6 +104,33 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	if err != nil {
 		return errors.Wrap(err, "could not create watch on WindowsMachineConfig objects")
 	}
+	// Watch for the machine objects
+	windowsOSLabel := "node.openshift.io/os_id"
```

Comment:
Not sure if we want to use a `node.openshift.io` for this. I prefer using `machine.openshift.io/os_id` (value will be `Windows` or `windows.machine.openshift.io` (value will be `True`)